sense/downstream_tasks/nn_utils.py

```diff
@@ -114,7 +115,15 @@ def load_weights(checkpoint_path: str):
     """
     try:
         return torch.load(checkpoint_path, map_location='cpu')
-    except:
-        raise Exception('ERROR - Weights file missing: {}. To download, please go to '
-                        'https://20bn.com/licensing/sdk/evaluation and follow the '
-                        'instructions.'.format(checkpoint_path))
+    except FileNotFoundError:
+        root_dir = Path(__file__).parent.parent.parent
+        exec_dir = Path(os.path.dirname(os.path.realpath("__main__")))
```

Comment:
I think recently we have replaced all usages of `Path` with their `os.path` counterparts. Until we've agreed on a project-wide convention, we should probably stick to that :grimacing:

Reply:
Idea for the `.parent.parent.parent` issue: Could we walk up the directory tree until we find the `sense` package?

Reply:
I was hesitant on the `.parent.parent.parent` as well. One way I can think of is to have a `utils.py` file in a fixed location in the project where we can store a `ROOT_DIR` const or a function.

We could also walk up the directory tree until sense, but how? StackOverflow did not answer this question on my quick search :laughing: 

Your second thought is very interesting! Let's see how @guillaumebrg thinks as well.

Reply:
> We could also walk up the directory tree until sense, but how? StackOverflow did not answer this question on my quick search laughing

The code snippet above is how :wink:

Reply:
@nahuakang @corneliusboehm, I was about to make a small change to the PR and ended up getting carried away... :) 
My proposal is here: https://github.com/TwentyBN/sense/pull/90
Let me know what you think